package.json

```diff
@@ -66,5 +66,6 @@
     "ts-jest": "^29.1.0",
     "tsup": "^8.0.2",
     "typescript": "^5.0.4"
-  }
+  },
+  "packageManager": "yarn@1.22.22"
```

Comment:
Side change: this helps Node’s [corepack](https://nodejs.org/api/corepack.html) to pick the right version of yarn. Otherwise, `yarn install` may pick v4 and break the lockfile.

Reply:
This _should_ be done independently but that's ok.

Reply:
Yeah I agree that this change is unrelated, so should have been in another PR. It was one of those borderline sidecar changes that could be tolerable, so I thought I'd sneak it in 😅